src/MainWindow.xaml.cs

```diff
@@ -27,5 +28,7 @@ private void MainWindow_Closed(object sender, WindowEventArgs args)
     {
         Application.Current.GetService<IExtensionService>().SignalStopExtensionsAsync();
         TelemetryFactory.Get<ITelemetry>().Log("DevHome_MainWindow_Closed_Event", LogLevel.Critical, new DevHomeClosedEvent(mainWindowCreated));
+        Log.Information("Terminating via MainWindow_Closed.");

```

Comment:
Reflections are amazing: 

          MethodBase m = MethodBase.GetCurrentMethod();
          Console.WriteLine("Executing {0}.{1}",  m.ReflectedType.Name, m.Name);

now the method name isn't hard coded.

Reply:
Agree with the sentiment, but I think that's a bit overkill for this instance and unnecessary code execution. I just wanted to log the source of termination, and the hard coded string is fine in this case. Easy to read, simple, and not too offensive.